recursive-list-main/src/components/folder/folder.component.tsx

```diff
@@ -0,0 +1,46 @@
+import { Box, Link, Paper, Typography } from "@material-ui/core";
+import { Link as RouterLink } from "react-router-dom";
+import { useFolderStyles } from "./folder.styles";
+
+import { List } from "components/list/list.component";
```

Comment:
move interface to .interfaces.ts file and import from there instead of importing from component